README.md

```diff
@@ -12,7 +12,7 @@ This application does not use state management, but uses the [http](https://pub.
 
 ## HTTP Package
 
-In order to use the HTTP package, I first needed to download it and import it at the top of my file. All HTTP requests require a url. For example, this is the url for getting the info from the camera. 
+In order to use the http package, I first needed to download it and import it at the top of my file. All HTTP requests require a url. For example, this is the url for getting the info from the camera. 
```

Comment:
I agree that we should change it here.  As mentioned above, we should be consistent in the use of capitalization of HTTP.  We should also change it above.